### PR TITLE
Add scanStream command

### PR DESCRIPTION
**Pull request by BastienAr**
My proposal to answer #362, first with `scanStream` command.

### CODE REVIEW

File: .jsbeautifyrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "eol": "\n",
+  "indent_char": " ",
+  "indent_size": 2,
+  "indent_with_tabs": false,
+  "brace_style": "none,preserve-inline",
+  "max_preserve_newlines": 2,
+  "wrap_line_length": 100,
```
Comment (stipsan) [inline .jsbeautifyrc]: This project is using prettier 😄  and have different line width defaults (80) so I think adding this file could cause conflicts for people who have both formatters installed in their IDE 😨

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,5167 @@
+Arguments:
```
Comment (stipsan) [inline yarn-error.log]: oops! 🙈

**Comment by stipsan** [comment]
Oh and don't worry too much about what codeclimate says, it's more a guideline rather than rules and sometimes it's better to change the codeclimate settings rather than refactoring the code. 
That is why I set it as "not required" on PRs, allowing us to merge even if it's red 😄

File: yarn-error.log (hunk 2 above)
Comment (BastienAr) [reply yarn-error.log]: 🙈@#$!!

File: .jsbeautifyrc (hunk 1 above)
Comment (BastienAr) [reply .jsbeautifyrc]: oops, sorry for this one, forgot to remove the file

**Comment by stipsan** [comment]
Awesome stuff @BastienAr, I'll merge this and prepare a release 👍 🎉

**Comment by BastienAr** [comment]
Cool, I will implement the others scan/stream and push another pr

**Comment by stipsan** [comment]
Brilliant! 👌

**Pull request merged**